media/css/sandstone/lib.less

```diff
@@ -551,3 +551,10 @@
                 fade(@pageBgColor, 0%) 320px, @auroraStartColor 380px);
     }
 }
+
+.sticky() {
```

Comment:
We should probably provide an equivalent mixin for this in Sass for Pebbles too?